lib/elixir/unicode/tokenizer.ex

```diff
@@ -215,12 +242,30 @@ defmodule String.Tokenizer do
        |> Enum.reduce(bottom, &ScriptSet.union/2)}
     end
 
+  # add our custom normalizations
+  codepoints_to_mask =
+    for {from, to} <- normalizations, reduce: codepoints_to_mask do
+      acc ->
+        ss = ScriptSet.union(Map.get(acc, from, top), Map.get(acc, to, top))
+        Map.put(acc, from, ss)
```

Comment:
This line has no effect so I assume it is not used?

Reply:
Oh, I think it should be used but there is no easy way to test it because our only from today is top!